evap/contributor/tests/test_views.py

```diff
@@ -284,3 +285,17 @@ def test_display_request_buttons(self):
         page = self.app.get(self.url, user=self.responsible)
         self.assertEqual(page.body.decode().count("Request changes"), 0)
         self.assertEqual(page.body.decode().count("Request creation of new account"), 2)
+
+
+class TestContributorResultsExportView(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        result = create_evaluation_with_responsible_and_editor()
+        cls.url = "/contributor/export"
```

Comment:
We usually lookup urls via their name as defined in `urls.py` (in this case `evap/contributor/urls.py`):
```suggestion
        cls.url = reverse("contributor:export")
```